packages/core/examples/volumeBasicWadoUri/index.ts

```diff
@@ -0,0 +1,150 @@
+import {
+  RenderingEngine,
+  Types,
+  Enums,
+  volumeLoader,
+  imageLoader,
+} from '@cornerstonejs/core';
+import {
+  initDemo,
+  setTitleAndDescription,
+  setCtTransferFunctionForVolumeActor,
+} from '../../../../utils/demo/helpers';
+import instanceUIDs from './instanceUIDs.js';
```

Comment:
It was giving me error in compile for this `instanceUIDs.js`, so I had to convert that to .ts